Modules/_sqlite/module.c

```diff
@@ -338,6 +344,23 @@ do {                                           \
     }                                          \
 } while (0)
 
+#define ADD_STRING(dict, key, val)                   \
+do {                                                 \
+    if (PyDict_SetItemString(dict, key, val) < 0) {  \
+        Py_DECREF(val);                              \
+        goto error;                                  \
+    }                                                \
+} while (0)
+
+#define ADD_EXCEPTION(dict, name, exc, base)                    \
```

Comment:
Since we can pass Module instead of Dict,
I 'd like to suggest to replace to use PyModule_AddObject instead PyDict_SetItemString